website/cue/reference/components/sources.cue

```diff
@@ -126,6 +126,24 @@ components: sources: [Name=string]: {
 									}
 								}
 							}
+							length_delimited: {
+								description:   "Options for `length_delimited` framing."
+								required:      false
+								common:        false
+								relevant_when: "method = `length_delimited`"
+								type: object: options: {
+									max_length: {
+										description: "The maximum frame length limit. A connection that sends a frame larger than `max_frame_length` will be closed."
```

Comment:
From Vector's perspective, do we really kill the stream or we just return an error?

Reply:
We close the connection. That's why I specified it. At least that is my experience. And also it makes sense. LengthDelimitedCodec reads the header, if the size is bigger it is not going to skip it, it would be nice, but it doesn't.

Reply:
Hmm, yeah, I agree, it would be nice if it could just "fast forward" over the problematic frame but I also see that's not the way that tokio_util::codec::length_delimited::LengthDelimitedCodec works.

Reply:
Yep, there's not really a way of doing it. It could read in small chunks if you want to fast-forward. But for that it would require our own codec instead of just tokio's.